extensions/sql-migration/src/wizard/loginSelectorPage.ts

```diff
@@ -522,6 +540,10 @@ export class LoginSelectorPage extends MigrationWizardPage {
 		return table;
 	}
 
+	private _isAllSelectedLoginsValidated(): boolean {
```

Comment:
nit: could be faster with !array.some(!validatedLogins.has(login))

Reply:
browsed through and verified both are equivalent on performance side
Best case: Both methods can short-circuit and stop early if they find a disqualifying element early in the array.
Worst case: Both methods will iterate through the entire array if all elements satisfy the condition.